includes/discovery/sensors/temperature/sentry4.inc.php

```diff
@@ -22,32 +22,36 @@
 $divisor    = '10';
 $multiplier = '1';
 if ($oids) {
+    echo 'ServerTech Sentry4 Temperature ';
     $sentry_temp_scale = snmp_get($device, 'st4TempSensorScale.0', '-Ovq', 'Sentry4-MIB');
-    if ($sentry_temp_scale == 'fahrenheit') {
-        $user_func = 'convert_to_celsius';
-    } else {
-        $user_func = null;
```

Comment:
Why did you delete this?  It is required if the data outputs farenheit.

Reply:
In my librenms instance, limit values not converted although I passed the user_func to discover_sensor. So I need to convert low/high limit values manually. Just to be consistent I convert current data value manually too. Also passing convert_to_celsius function to discover_sensor does not work properly but fahrenheit_to_celcius works. I think these two bugs should be addressed separately.